components/org.wso2.carbon.identity.oauth.ui/src/main/java/org/wso2/carbon/identity/oauth/ui/client/OAuthAdminClient.java

```diff
@@ -179,6 +181,22 @@ public OAuthTokenExpiryTimeDTO getOAuthTokenExpiryTimeDTO() throws RemoteExcepti
         return scopeValidators;
     }
 
+    /**
+     * Get the registered oauth token types from OAuth server configuration file.
+     *
+     * @return List of supported oauth token types
+     * @throws RemoteException exception occured during remote call
+     */
+    public String[] getSupportedTokenTypes() throws RemoteException {
```

Comment:
Add new line after the method declaration

Reply:
fixed with 0c8dfb39735118757027bcb9748516cebd8dd8c1